api/src/services/notifications.ts

```diff
@@ -16,13 +17,22 @@ export class NotificationsService extends ItemsService {
 	}
 
 	async createOne(data: Partial<Notification>, opts?: MutationOptions): Promise<PrimaryKey> {
-		await this.sendEmail(data);
+		try {
+			await this.sendEmail(data);
+		} catch (error: any) {
+			logger.error(error.message);
+		}
```

Comment:
We should swap this with the `super.createOne(data, opts)` call to solve for item 3 from https://github.com/directus/directus/issues/14100#issue-1284309285

Reply:
So put the `super.createOne(data, opts)` into the try/catch?

Reply:
The super call should happen before the try catch block altogether. Also, the try/catch of the send mail bit should happen in sendMail itself, otherwise the same problem will happen everywhere else we send an email 👍🏻